packages/react-hooks/src/useId.ts

```diff
@@ -0,0 +1,17 @@
+import { getId } from '@uifabric/utilities';
+import { useConst } from './useConst';
+
+/**
+ * Hook to generate a unique ID in the global scope (spanning across duplicate copies of the same library).
+ *
```

Comment:
Is it our goal to provide these hooks for anyone/everyone? Feels like this is primarily for our own benefit -- can we mark it `@internal` to discourage people from depending on it explicitly?

Reply:
I would think it's generalized enough though.

Reply:
I love this, btw!